test-case-reporting/types/test-case-reporting.d.ts

```diff
@@ -61,16 +61,27 @@ declare module 'test-case-reporting' {
     durationMs: number,
   }
 
-  export interface TravisReport {
-    successCount: number,
-    failedCount: number,
-    skippedCount: number,
-    results: Array<Result>,
+  export interface TestResultReport {
+    summary: TrrSummary,
+    browsers: TrrBrowsers,
   }
 
-  export interface Result {
-    name: string,
-    status: TestStatus,
-    timeMs: number,
+  export interface TrrSummary {
```

Comment:
How will this data be used? Does it match up with any of our DB columns?

Reply:
Oh, you're right -- I added that thinking it might be useful for the web view, but the web view will be fetching from the database, so this not used.

I'll remove it and let you know when done.

Reply:
Done.